.github/workflows/reusable-e2e.yml

```diff
@@ -280,17 +324,25 @@ jobs:
           aws configure set aws_session_token $AWS_SESSION_TOKEN && \
           aws configure set default.region $AWS_REGION
 
-      - name: Pull down In-app-messaging AWS environments
+      - name: Pull down Gen1 In-app-messaging AWS environments
+        if: ${{ matrix.version == 'gen1' }}
         run: yarn environments in-app-messaging pull
 
+      - name: Pull down Gen2 In-app-messaging AWS environments
+        if: ${{ matrix.version == 'gen2' }}
+        run: yarn environments in-app-messaging pull -v gen2
+
       - name: Delete AWS profile
         run: rm -rf ~/.aws
 
       - name: Build ${{ matrix.example }} example
         run: yarn ${{ matrix.example }}-example build
+        env:
+          VERSION: ${{ matrix.version }}
+          VITE_VERSION: ${{ matrix.version }}
 
       - name: Start ${{ matrix.example }} example
-        run: yarn ${{ matrix.example }}-example start & npx wait-on -c waitOnConfig.json -t 20000 http-get://localhost:3000/ui/components/authenticator/sign-in-with-username
+        run: yarn ${{ matrix.example }}-example start & npx wait-on -c waitOnConfig.json -t 20000 http-get://localhost:3000/ui/components/authenticator/sign-in-with-email
```

Comment:
Can you provide a screenshot in the PR description showing passing Android and iOS e2e test runs?

Reply:
Sure, I will run against Android and iOS e2e tests